pontoon/base/static/js/main.js

```diff
@@ -143,6 +143,38 @@ $(function () {
         ga('send', 'event', 'ajax', 'request', settings.url);
     });
 
+    /*
+     * Display Pontoon Add-On promotion, if:
+     *
+     * - Pontoon Add-On not installed
+     * - Page loaded on Firefox or Chrome (add-on not available for other browsers)
+     */
+    var isFirefox = navigator.userAgent.indexOf('Firefox') !== -1;
+    var isChrome = navigator.userAgent.indexOf('Chrome') !== -1;
+    var downloadLink = $('#addon-promotion').find('.get');
+    setTimeout(function () {
+        if (!window.PontoonAddon || !window.PontoonAddon.installed) {
```

Comment:
You also need to react on when the extension is installed after the page is loaded, something like
```js
window.addEventListener('message', (e) => {
  const data = JSON.parse(e.data);
  if (data._type === 'PontoonAddonInfo') {
    const pontoonAddonInfo = data.value;
    if (pontoonAddonInfo.installed) {
      $('body').removeClass('addon-promotion-active');
    }
  }
});
```

Reply:
Yup, that's on my TODO list.